SPECS/dracut/add-livenet-download-service.patch

```diff
@@ -0,0 +1,147 @@
+From 2888695d6e1e03f727fe45d6e38fc71f7b5be123 Mon Sep 17 00:00:00 2001
```

Comment:
This change should probably be split off into a separate PR so that it can reviewed separately from the MIC PXE feature.

Reply:
Moved out.